pkgs/development/python-modules/pkgconfig/default.nix

```diff
@@ -40,6 +40,8 @@ buildPythonPackage rec {
 
   nativeBuildInputs = [ poetry-core ];
 
+  # TODO: figure out why propagatedNativeBuildInputs is not adding pkg-config to PATH when pkgconfig is used in nativeBuildInputs
```

Comment:
unless there's logic to capture the path, a nativeBuildInput probably shouldn't be exposing.

We should instead be patching where pkgconfig gets called.

Reply:
There is already such a patch but the pkg-config wrapper/setup hook always sets PKG_CONFIG to ``pkg-config`` which makes it pretty useless https://github.com/NixOS/nixpkgs/blob/master/pkgs/development/python-modules/pkgconfig/executable.patch

Reply:
unless I missed something, that appears to be no-op

Reply:
Yes. So we want to hardcode the path to ``${pkg-config}/bin/pkg-config``?